### PR TITLE
Remove unused classes and methods from API

### PR DESCRIPTION
**Pull request by hokwaichan**
Removed unused test

# Ticket Link

[Groupings-1540](https://uhawaii.atlassian.net/browse/GROUPINGS-1540)

# List of squashed commits

-  Removed getGroupingTest

# Test Checklist

- [x] Exhibits Clear Code Structure:
- [x] Project Unit Tests Passed:
- [x] Project Integration Tests Passed:
- [x] Executes Expected Functionality:
- [x] Tested For Incorrect/Unexpected Inputs:

### CODE REVIEW

File: src/main/java/edu/hawaii/its/api/exception/AccessDeniedException.java [hunk 1]
```diff
@@ -1,8 +1,6 @@
 package edu.hawaii.its.api.exception;
 
 public class AccessDeniedException extends RuntimeException {
-    private static final long serialVersionUID = 1L;
```
Comment (JorWo) [inline src/main/java/edu/hawaii/its/api/exception/AccessDeniedException.java:4]: I think we actually need this line.

File: src/test/java/edu/hawaii/its/api/util/JsonUtilTest.java [hunk 2]
```diff
@@ -13,16 +13,18 @@ public class JsonUtilTest {
...
-        assertEquals(sd0.getDescription(), sd1.getDescription());
-        assertEquals(sd0.isSynced(), sd1.isSynced());
-        assertEquals(sd0.isHidden(), sd1.isHidden());
-        assertEquals(sd0.getTooltip(), sd1.getTooltip());
+        assertEquals(sd0.getUhUuid(), sd1.getUhUuid());
+        assertEquals(sd0.getUsername(), sd1.getUsername());
+        assertEquals(sd0.getFirstName(), sd1.getFirstName());
+        assertEquals(sd0.getLastName(), sd1.getLastName());
+        assertEquals(sd0.getAttributes(), sd1.getAttributes());
+        assertEquals(sd0.getClass(), sd1.getClass());
```
Comment (JorWo) [inline src/test/java/edu/hawaii/its/api/util/JsonUtilTest.java:27]: Be sure to rename the variables:
- sd0 -> person0
- sd1 -> person1

**Review by JorWo — APPROVED** [review]
Great work!

**Review by mhodgesatuh — APPROVED** [review]
That's an impressive amount of cleanup.

**Pull request merged**